client/my-sites/theme/theme-download-card/index.jsx

```diff
@@ -11,16 +11,15 @@ import Gridicon from 'gridicons';
 import Button from 'components/button';
 import Card from 'components/card';
 
-const ThemeDownloadCard = React.createClass( {
+class ThemeDownloadCard extends React.Component {
```

Comment:
Yay for the ES6ification! Since this doesn't have any internal state AFAICS, you should have it extend `React.PureComponent` for better perf (or turn it into a stateless functional component).